internal/core/unittest/test_expr.cpp

```diff
@@ -1513,7 +1513,7 @@ TEST(Expr, TestExprs) {
         }
     };
     auto test_case = [&](int n) {
-        auto expr = build_expr(TermExprImpl, n);
+        auto expr = build_expr(UnaryRangeExpr, n);
```

Comment:
why change the OpType?
add a new testcase ?

Reply:
> why change the OpType? add a new testcase ?

can ignore this